src/renderer/Voice.tsx

```diff
@@ -711,6 +718,14 @@ const Voice: React.FC<VoiceProps> = function ({ t, error: initialError }: VoiceP
 				}
 				return { ...old };
 			});
+			setPlayersThatHaveDiedThisGame((prevState) => {
+				for (const player of gameState.players) {
+					if ((player.isDead || player.disconnected) && !prevState.includes(player.ptr)) {
+						prevState.push(player.ptr);
+					}
+				}
+				return [...prevState];
+			});
```

Comment:
I might be missing something but couldn't you use the above `otherDead` variable? It appears to be set at the time time and to the same values.